demo/nodejs/README.md

```diff
@@ -51,18 +51,17 @@ by Orca to your device's speakers.
 Execute the following:
 
 ```console
-orca-streaming-demo --access_key ${ACCESS_KEY} --language ${LANGUAGE} --gender ${GENDER} --text_to_stream ${TEXT}
+orca-streaming-demo --access_key ${ACCESS_KEY} --model_file_path ${MODEL_FILE_PATH} --text_to_stream ${TEXT}
 ```
 
-Replace `${ACCESS_KEY}` with your `AccessKey` obtained from Picovoice Console, `${LANGUAGE}` and `${GENDER}` with the language and gender you would like to run the demo in (available languages are `en`, `es`, `de`, `fr`, `ko`, `ja`, `it`, `pt`, and available genders are `male` and `female`), and `${TEXT}` with your text to be
-streamed to Orca.
+Replace `${ACCESS_KEY}` with your `AccessKey` obtained from Picovoice Console, `${MODEL_FILE_PATH}` with a path to any of the model files available under [lib/common](../../lib/common), and `${TEXT}` with your text to be streamed to Orca.
```

Comment:
I think this readme ends up on npm, so you can't use relative link paths like `../../lib/common`. Same for python. You'll have to use full URL